org-code-javabuilder/lib/src/test/java/org/code/javabuilder/ProjectDataTest.java

```diff
@@ -0,0 +1,89 @@
+package org.code.javabuilder;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import org.code.protocol.InternalErrorKey;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+// TODO: Test validation JSON when available
```

Comment:
what info do you need for validation json? You can get sample validation json from all the things java lab level 12

Reply:
Oh good to know! Wasn't sure if I could get validation data from a level yet, so I'll add test cases for that.

Reply:
I'll go ahead and remove this TODO for now - I confirmed locally that validation JSON is present, but we can defer testing against expected output until we need to pull out validation code separately from source code.